samples/bluetooth/ble_hids_keyboard/README.rst

```diff
@@ -83,7 +84,6 @@ Button 0:
 
 Button 1:
    Simulates the Shift key.
-   Keep the button pressed while resetting the board to delete bonding information for all peers stored on the device.
```

Comment:
I think it would be of value to mention this somewhere else, if not here.

Reply:
As the sample is documented now, we have only one section for user interface.
It could be placed in test section but I think it still should be written her for clarity.

The extra line describing how to delete bond should be kept here. Maybe put it in bracket or in any way indicate that this additional functionality.

Reply:
My mistake, I didn't intend to delete it.